src/types/packet.rs

```diff
@@ -21,7 +21,8 @@ pub struct Nack {
 
 pub enum NackType {
     ErrorInRouting(NodeId), // contains id of not neighbor
-    Dropped(),
+    DestinationIsNotAServer,
```

Comment:
I think that this should be called 'DestinationIsDrone' as the other one, since clients are also possibile destination. Other than that seems good to me, we'll vote on this tomorrow.

Reply:
Fixed